src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/opentelemetry/otel-adot-log-ingestion-serverless.mdx

```diff
@@ -0,0 +1,191 @@
+---
+title: OTEL log ingestion solution for ADOT lambda layer collector
+metaDescription: How to instrument your serverless applications on AWS Lambda with ADOT solution in New Relic.
+freshnessValidatedDate: never
+---
+You can now monitor AWS serverless application that sends OTEL log events data from CloudWatch Logs to the New Relic UI.
+You can configure the ADOT lambda layer collector to successfully forward OTEL application logs with all necessary service and application context. 
+This includes ensuring that logs contain metadata such as service name, version, environment, and any other relevant attributes that can aid in observability and troubleshooting.
+
+
+## Pre-requisites
+
+* Make sure you have a <InlinePopover type="licenseKey" />.
+
+
+
+## Procedure
+
+The following setup shows one approach for configuring environment variables to your AWS Lambda application. You can also configure them from the [Functions](https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html) page.
```

Comment:
Make it clear that you're linking someone out to an external resource (which we usually try to avoid). For example:

"You can also configure them on the AWS functions page. For more information, [see the Amazon Lambda environment variables doc](https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html)."

We don't like linking to external resources, because we have no control over the content in it or even how long the link will stay valid.